Sources/Common/Background Queue/Type/QueueTask.swift

```diff
@@ -52,7 +52,7 @@ public struct QueueTask: Codable, AutoLenses, Equatable {
 internal extension QueueTask {
     static var random: QueueTask {
         QueueTask(storageId: String.random,
-                  type: QueueTaskType.identifyProfile.rawValue,
+                  type: String.random,
```

Comment:
doesn't seem to be a test file so why change the the profile identifier to a random string?